tools/http3_test/tests/httpbin_tests.rs

```diff
@@ -68,6 +68,20 @@ mod httpbin_tests {
         }
     }
 
+    fn extra_headers() -> Option<String> {
+        match std::env::var_os("EXTRA_HEADERS") {
+            Some(val) => Some(val.into_string().unwrap()),
+            None => None,
+        }
+    }
```

Comment:
I guess most tests would want the the parsed key-value map from the environment variable, so perhaps this function could do all of that (i.e. pull in the `serde_json` calls in headers() test below) and then just return a `Some(serde_json::Map<String, serde_json::Value>)`

Reply:
sure, I'll do it.